modules/apps/object/object-service/src/main/java/com/liferay/object/internal/security/permission/resource/ObjectEntryFolderPortletResourcePermissionWrapper.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.internal.security.permission.resource;
+
+import com.liferay.object.constants.ObjectEntryFolderConstants;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.security.permission.PermissionChecker;
+import com.liferay.portal.kernel.security.permission.resource.BasePortletResourcePermissionWrapper;
+import com.liferay.portal.kernel.security.permission.resource.PortletResourcePermission;
+import com.liferay.portal.kernel.security.permission.resource.PortletResourcePermissionFactory;
+import com.liferay.portal.kernel.security.permission.resource.PortletResourcePermissionLogic;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Feliphe Marinho
+ * @author Mario Gomes
+ */
+@Component(
+	property = "resource.name=" + ObjectEntryFolderConstants.RESOURCE_NAME,
+	service = PortletResourcePermission.class
+)
+public class ObjectEntryFolderPortletResourcePermissionWrapper
+	extends BasePortletResourcePermissionWrapper {
+
+	@Override
+	protected PortletResourcePermission doGetPortletResourcePermission() {
+		return PortletResourcePermissionFactory.create(
+			ObjectEntryFolderConstants.RESOURCE_NAME,
+			new ObjectEntryFolderPortletResourcePermissionWrapper.
+				ObjectEntryFolderPortletResourcePermissionLogic());
+	}
```

Comment:
Why are you using
`new ObjectEntryFolderPortletResourcePermissionWrapper.ObjectEntryFolderPortletResourcePermissionLogic()`
instead of
`new ObjectEntryFolderPortletResourcePermissionLogic()` ?

I saw that in ObjectPortletResourcePermissionWrapper:
```
protected PortletResourcePermission doGetPortletResourcePermission() {
    return PortletResourcePermissionFactory.create(
    ObjectConstants.RESOURCE_NAME,
    new ObjectPortletResourcePermissionLogic());
}
```